renku/ui/cli/config.py

```diff
@@ -97,13 +97,19 @@
 | ``lfs_threshold``              | Threshold file size below which     | ``100kb`` |
 |                                | files are not added to git LFS      |           |
 +--------------------------------+-------------------------------------+-----------+
-| ``show_lfs_message``           | Whether to show messages about      | ``True``  |
+| ``show_lfs_message``           | Whether to show messages about      | ``true``  |
 |                                | files being added to git LFS or not |           |
 +--------------------------------+-------------------------------------+-----------+
-| ``show_login_warning``         | Whether to warn when logging in to  | ``True``  |
+| ``show_login_warning``         | Whether to warn when logging in to  | ``true``  |
```

Comment:
Same as above; I've just realized that I've made an error when reading this config in https://github.com/SwissDataScienceCenter/renku-python/blob/fff20ba40bed3a3dcc5a0c1620f0ee9cc6570161/renku/core/login.py#L67. It return a str not a bool.